### PR TITLE
[IIIF-807] Add code style check to Travis CI

### PR DESCRIPTION
**Pull request by markmatney**
I figure we might as well try out black and see what we think.

I'd like to see the Travis build fail with the new build config, and then I'll add an additional commit with all Python files reformatted.

### CODE REVIEW

**Comment by markmatney** [comment]
Black supports Python 3.6 and later, so I dropped the 3.5 check from our Travis build. I think this is fine, since our three primary users have confirmed to me that they're all on 3.7.

File: README.md [hunk 1]
```diff
@@ -117,12 +117,16 @@ It is recommended that developers create a virtual environment for local Python
...
     python3 -m venv venv_festerize
     . venv_festerize/bin/activate
-    pip install -e . pytest
+    pip install -e . black pytest
 
 To run the tests:
 
     pytest
 
+Before committing, make sure you format all the Python source files:
```
Comment (ksclarke) [inline README.md:126]: Might be nice to supply a pre-push git hook?

Comment (markmatney) [reply README.md]: Good idea. I created a pre-push hook so a dev could keep the files in whatever state is most appropriate for them until they're ready to share

Comment (ksclarke) [reply README.md]: Yeah, that's better.

**Pull request merged**